libs/content/workspace/src/workspace.rs

```diff
@@ -142,7 +194,17 @@ impl Workspace {
             focused_parent: None,
             last_touch_event: None,
             out: output,
-        }
+        };
+
+        let last_open_tabs = ws.cfg.data.read().unwrap().last_open_tabs.clone();
```

Comment:
from my last review: 

> I picture an API that let's you read a one-off setting, ergonomic functions for mutating the settings that kickoff the file update. These functions should take care of the lock for me, and should be named in a way that's easy to discover.

Reply:
I basically expect this to be `ws.cfg.get_tabs()` handling ownership and concurrency concerns inside the API